public/stylesheets/style.css

```diff
@@ -1,312 +1,316 @@
+  padding: 3px;
 }
 a:hover {
-	color: whitesmoke;
+  color: whitesmoke;
 }
 .btn-nav {
-	color: whitesmoke;
-	border-radius: 6px;
-	padding: 3px 6px;
-	font-size: 0.8rem;
-	text-decoration: none;
-	color: whitesmoke;
-	font-weight: bold;
-	transition: color 0.1s;
-	border: none;
-	border: 0px;
-	background-color: rgb(0, 0, 139, 0.8);
-	margin: 5px 20px;
+  color: whitesmoke;
+  border-radius: 6px;
+  padding: 3px 6px;
+  font-size: 0.8rem;
+  text-decoration: none;
+  color: whitesmoke;
+  font-weight: bold;
+  transition: color 0.1s;
+  border: none;
+  border: 0px;
+  background-color: rgb(0, 0, 139, 0.8);
+  margin: 5px 20px;
 }
 .btn-nav:hover {
-	color: skyblue;
+  color: skyblue;
 }
 footer {
-	text-align: center;
-	color: darkblue;
-	align-content: flex-end;
-	background-color: rgba(158, 205, 247, 0.8);
-	position: sticky;
-	top: 100vh;
-	padding: 10px;
-	z-index: -1;
+  text-align: center;
+  color: darkblue;
+  align-content: flex-end;
+  background-color: rgba(158, 205, 247, 0.8);
+  position: sticky;
+  top: 100vh;
+  padding: 10px;
+  z-index: -1;
 }
 .blog-title {
-	background-color: rgb(245, 245, 245, 0.8);
-	margin: 10px;
-	border-radius: 15px;
-	color: darkblue;
-	text-align: center;
-	font-weight: 700;
-	padding: 2rem;
-	font-size: 2.2rem;
+  background-color: rgb(245, 245, 245, 0.8);
+  margin: 10px;
+  border-radius: 15px;
+  color: darkblue;
+  text-align: center;
+  font-weight: 700;
+  padding: 2rem;
+  font-size: 2.2rem;
 }
 .sub-heading {
-	font-size: 2rem;
-	text-align: center;
-	color: darkblue;
-	margin: 0 auto;
-	max-width: inherit;
-	border-radius: 7px;
-	padding: 10px;
-	font-weight: 700;
-	background-color: rgb(245, 245, 245, 0.8);
+  font-size: 2rem;
+  text-align: center;
+  color: darkblue;
+  margin: 0 auto;
+  max-width: inherit;
+  border-radius: 7px;
+  padding: 10px;
+  font-weight: 700;
+  background-color: rgb(245, 245, 245, 0.8);
+}
+
+.comments {
+  font-size: 16px;
+  text-align: center;
+  color: darkblue;
+  border-radius: 7px;
+  width: 50%;
+  padding: 10px;
+  background-color: rgb(245, 245, 245, 0.8);
+  margin-top: 10px;
+  margin-left: 25%;
 }
 .blog-heading {
-	font-size: 1.6rem;
-	text-align: center;
-	color: darkblue;
-	background-color: rgb(192, 192, 192, 0.8);
-	margin: 25px 20px;
-	max-width: inherit;
-	border-radius: 7px;
-	padding: 10px;
-	font-weight: 700;
+  font-size: 1.6rem;
+  text-align: center;
+  color: darkblue;
+  background-color: rgb(192, 192, 192, 0.8);
+  margin: 25px 20px;
+  max-width: inherit;
+  border-radius: 7px;
+  padding: 10px;
+  font-weight: 700;
 }
 .form {
-	background-color: rgb(192, 192, 192, 0.8);
-	margin: 0 auto;
-	max-width: 275px;
-	border-radius: 10px;
-	font-weight: 500;
-}
-.create-blog-form {
-	background-color: rgb(192, 192, 192, 0.8);
-	margin: 0 auto;
-	width: 40%;
-	border-radius: 10px;
-	font-weight: 500;
+  background-color: rgb(192, 192, 192, 0.8);
+  margin: 0 auto;
+  max-width: 275px;
+  border-radius: 10px;
+  font-weight: 500;
```

Comment:
we need to keep create blog form here

Reply:
I think I got trigger happy when I was resolving the conflict, are you able to add these back when you review?

Reply:
It should be back in there now.